### ISSUE TITLE
Surveys for competency questions

### ISSUE THREAD

**Issue by HendrikBorgelt** [nfdi4cat/ontology-development#1]
Hello at all,

as competency questions should not necessarily be formulated by a semantic expert, maybe it is best to invite M. Liebau and let students invested in the N4Cat teaching course, formulate some competency questions (especially those content-related questions/chemistry questions). But then we would need to prepare some slides and a template on how to write and submit competency questions (a simple Excel template which would be collected manually should be sufficient).

This should however be open for discussion.

best regards
Hendrik

**Comment by dalito** [comment]
> a template on how to write and submit competency questions (a simple Excel template...)

Alternatively we could use a customized issue template for this. Then the competency questions would land here directly. Also it would be possible to reference to them by issue number.

**Comment by markdoerr** [comment]
Issues might be already too complex for some - although I like the logic behind @dalito ' s suggestion.
Why not simply start with some questions as "templates" and ask the students to simply add more of them with the in-build text editor (which is as simple as clicking the "edit" button when viewing the questions) ?

(Almost no git knowledge required, just how to use a web-browser ;)

**Comment by HendrikBorgelt** [comment]
I agree with @markdoerr that issues might be too complex for "non-trained" researchers wanting to collaborate, at the same time I must remind, you that I intended this to be specifically for students who are taught at one of the lectures of NFDI4Cat.  So a short introduction on how to write an issue should be more than feasible.

I don't think that researchers just happily stumble over a git Repo and do not know how to participate.

My most immediate concern is that having 1 issue per competency question can result in thousands of issues (some due to reposting, the others just due to simple permutations) which we would need to maintain. If there is a simple way of "closing the issues" and therefore not much maintenance is required from our side, I would be happy to go for issues.

if so desired I can create the issue template.
